### DISCUSSION TITLE
How to add fields to the user model with EmailProvider

### DISCUSSION THREAD

**Discussion by Hamziss** [nextauthjs/next-auth#5441] [Help]
### Question 💬

so my question is pretty simple when using a build in OAuth service(google,twitter,...) i get it i can add field to user model fields by the returned fields from profile() function but what about EmailProvider how can i change the model when the user is created ?

PS: im using mongoDB adapter

### How to reproduce ☕️

```js
import EmailProvider from `next-auth/providers/email`
...
providers: [
  EmailProvider({
    server: process.env.EMAIL_SERVER,
    from: process.env.EMAIL_FROM,
    // maxAge: 24 * 60 * 60, // How long email links are valid for (default 24h)
  }),
],
...

```

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by eddyhdzg** [comment]
I don't see a profile function as in other providers 😔.
